lib/translators/object/ObjectsTranslator.ts

```diff
@@ -0,0 +1,225 @@
+import { HexBuffer } from '../../HexBuffer';
+import { W3Buffer } from '../../W3Buffer';
+
+interface Modification {
+    id: string,
+    type: any; // 'int' | 'real' | 'unreal' | 'string',
```

Comment:
Type here is a `string` since it is one of four string options.
This could be codified in an `enum` from TypeScript, but I think that would mean that a developer using WC3MapTranslator would need to have an enum file to import as well, which might be overkill.

We can use a type definition (see: https://angularfirst.com/typescript-string-enums/), defined right before `interface Modification`
```
type ModificationType = 'int' | 'real' | 'unreal' | 'string';

interface Modification {
    ...
    type: ModificationType
}
```

This way TypeScript will give an error if a developer tries to use anything other than the four allowed types.

Reply:
Done ;)